packages/shared/components/Select/Select.story.tsx

```diff
@@ -0,0 +1,108 @@
+/*
+Copyright 2019 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React from 'react';
+import styled from 'styled-components';
+import { Flex, Box, LabelInput } from 'design';
+import Select, { DarkStyledSelect } from '../Select';
+
+export default {
+  title: 'Shared/Select',
```

Comment:
Since its just one file for now, rename to `Shared`, and rename `SelectTypes` as `Selects` so there is no nesting